<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferWindow</class>
 <widget class="QWidget" name="TransferWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TransferWindow {border-image: url(:atm-frontend/bg.png) 0 0 0 0 stretch stretch;}
QWidget { color: white;}</string>
  </property>
  <widget class="QLabel" name="labelTransferTitle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tilisiirto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccountId">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tili ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBalance">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Käytettävissä:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelReceiverId">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vastaanottajan tilinumero:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditReceiverId">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelAmount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Summa:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAmount">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>LOPETA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>HYVÄKSY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorMessage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
